Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
```

Comment:
That's down to @mantognini, then I guess, if we move any further on this. He probably doesn't want to remove the block entirely as I've done, but he may be able to wrap it in a command-flag we can use instead where when that flag is triggered the block is ` ifndef `-ed out or similar.